{"name": "test_edit_button_in_view_failure_reason", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <TestCases.Package_entity_management.TC_failure_reason.Test_Failure_Reason testMethod=test_edit_button_in_view_failure_reason>\n\n    @allure.description(\"Should display the edit failure reason popup window \")\n    @allure.severity(severity_level=\"CRITICAL\")\n    @pytest.mark.order(7_07)\n    def test_edit_button_in_view_failure_reason(self):\n        self.failure_reason =failure_reason(self.driver)\n>       self.failure_reason.edit_failure_reason_button()\n\nTestCases/Package_entity_management/TC_failure_reason.py:73: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage_objects/entity_Management/failure_reason.py:110: in edit_failure_reason_button\n    if WebDriverWait(self.driver, 50).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"0573377afe2cb9ad22e44783062930ba\")>\nmethod = <function title_contains.<locals>._predicate at 0x7fd057e30790>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../../.local/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException"}, "description": "Should display the edit failure reason popup window ", "start": 1678801537977, "stop": 1678801588303, "uuid": "b1486fdf-866f-49b4-a511-cc6cb1acf864", "historyId": "be486dd1d3f732c80b80388f86c8f5cd", "testCaseId": "db4de9229d3f275974fcd9383a76836f", "fullName": "Test_Suites.test_all_module.Test_Failure_Reason#test_edit_button_in_view_failure_reason", "labels": [{"name": "severity", "value": "CRITICAL"}, {"name": "tag", "value": "order(707)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "Test_Suites"}, {"name": "suite", "value": "test_all_module"}, {"name": "subSuite", "value": "Test_Failure_Reason"}, {"name": "host", "value": "sujith-HP-Compaq-Elite-8300-SFF"}, {"name": "thread", "value": "27332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_Suites.test_all_module"}]}